ffi/include/tvm/ffi/container/ndarray.h

```diff
@@ -110,6 +109,21 @@ inline size_t GetDataSize(const DLTensor& arr) {
   return GetDataSize(size, arr.dtype);
 }
 
+/*!
+ * \brief Infer the stride from shape
+ *
+ * \param shape the input Shape
+ * \return the inferred stride
+ */
+inline Shape InferStrideFromShape(Shape shape) {
+  size_t ndim = shape.size();
+  Array<int64_t> strides(ndim, 1);
```

Comment:
do not use array, instead, use https://github.com/apache/tvm/blob/main/ffi/include/tvm/ffi/container/shape.h#L71
details::MakeEmptyShape, then fill in the strides. 

do something like

```
int64_t stride = 1;

for  () {
   assign
}
```
We can also move this function to shape.h details::